branch-diff.js

```diff
@@ -148,7 +148,11 @@ function collect (repoPath, branch, startCommit, endRef) {
 module.exports = branchDiff
 
 if (require.main === module) {
-  let argv          = require('minimist')(process.argv.slice(2))
+  const config = {
+    boolean: ['version', 'size', 'group', 'patch-only', 'simple',
```

Comment:
I can't find any documentation for the `size` argument. Is that used anywhere?

Reply:
I can't find it either, I'm not sure why I added it, it's gone.